src/content/recipes/@emotion/styled.md

```diff
@@ -29,33 +29,51 @@ If you’d like to see the example code of this recipe, check out the [example r
 
 ![Completed Emotion example with theme switcher](https://user-images.githubusercontent.com/18172605/208312563-875ca3b0-e7bc-4401-a445-4553b48068ed.gif)
 
+## Install `@storybook/addon-styling`
+
+Add the `@storybook/addon-styling` package to your DevDependencies
+
+```shell
+yarn add -D @storybook/addon-styling
+```
+
+Then register with Storybook in `.storybook/main.js`.
+
+```js
+module.exports = {
+  stories: ['../stories/**/*.stories.mdx', '../stories/**/*.stories.@(js|jsx|ts|tsx)'],
+  addons: ['@storybook/addon-essentials', '@storybook/addon-styling'],
+};
+```
+
 ## How to setup `Global` styles
 
 UIs often have a set of global styles that are applied to every component like CSS resets, `font-size`, `font-family`, and colors.
 
-In Emotion, use the [`Global`](https://emotion.sh/docs/globals) to scope styles globally instead of locally (which is the library's default behavior).
+In Emotion, use the [`Global`](https://emotion.sh/docs/globals) component to scope styles globally instead of locally (which is the library's default behavior).
 
-Open `.storybook/preview.js` and create a `Global` component which includes a `font-family`. Then apply it to all stories via a [decorator](/docs/react/writing-stories/decorators).
+Open `.storybook/preview.js` and create a `GlobalStyle` component which includes a `font-family`. Then apply it to your stories with the [`withThemeFromJSXProvider`](https://github.com/storybookjs/addon-styling/blob/main/docs/api.md#withthemefromjsxprovider) decorator by adding it to the `decorators` array.
 
 ```js
 // .storybook/preview.js
-
+import { withThemeFromJSXProvider } from '@storybook/addon-styling';
 import { Global, css } from '@emotion/react';
 
-const withGlobalStyle = (Story) => (
-  <>
-    <Global
-      styles={css`
-        body {
-          font-family: 'Nunito Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
-        }
-      `}
-    />
-    <Story />
-  </>
+const GlobalStyle = () => (
+  <Global
+    styles={css`
+      body {
+        font-family: 'Nunito Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+      }
+    `}
+  />
 );
 
-export const decorators = [withGlobalStyle];
+export const decorators = [
+  withThemeFromJSXProvider({
+    GlobalStyles: GlobalStyle, // Adds your GlobalStyle component to all stories
```

Comment:
If you name the `GlobalStyle` component `GlobalStyles` instead, you can simplify this line to:

```suggestion
    GlobalStyles, // Adds your GlobalStyle component to all stories
```